vector/src/main/java/im/vector/app/features/home/HomeActivity.kt

```diff
@@ -229,6 +230,11 @@ class HomeActivity :
         if (args?.clearNotification == true) {
             notificationDrawerManager.clearAllEvents()
         }
+        if (args?.inviteNotificationRoomId != null) {
+            activeSessionHolder.getSafeActiveSession()?.permalinkService()?.createPermalink(args.inviteNotificationRoomId)?.let {
```

Comment:
what happens if the session isn't ready, is it something we need to worry about?

Reply:
It would be ready (done early in the application). Active session is optional because you might not be yet logged in, and the injection is done via dagger. I should have done this via view model TBH (there i'd have the session)